src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/OverflowMenu.component.tsx

```diff
@@ -60,37 +59,40 @@ export const OverflowMenuComponent = ({
                 component={
                     <FlyoutMenu dense maxWidth="250px">
                         <MenuItem
-                            label={i18n.t('View linked event')}
+                            label={i18n.t('View linked event') as string}
```

Comment:
Remove the `as string`. Remember to never add `as string` after the ` i18n.t(`) function